test/e2e/serial/tests/configuration.go

```diff
@@ -74,10 +74,10 @@ import (
 )
 
 /*
-MCP gets updated=false & updating=true in 2 cases:
- 1. mc is being updated with new configuration -> new CONFIG: all nodes associated to the pool will be updated
- 2. machine count is increasing: meaning new nodes joins to the pool by matching their roles to the machineConfigSelector.matchExpressions of the mcp.
-    so the same CONFIG on the pool but different machine count. This will trigger reboot on the new nodes thus mcp Updating contidtion will be true.
+	 MCP gets updated=false & updating=true in 2 cases:
+	  1. mc is being updated with new configuration -> new CONFIG: all nodes associated to the pool will be updated
+	  2. machine count is increasing: meaning new nodes joins to the pool by matching their roles to the machineConfigSelector.matchExpressions of the mcp.
+		 so the same CONFIG on the pool but different machine count. This will trigger reboot on the new nodes thus mcp Updating contidtion will be true.
```

Comment:
this seems unnecessary

Reply:
Totally unnecessary, probably happened from saving the file for some reason.